eZ/Publish/SPI/Persistence/Content/Type/Handler.php

```diff
@@ -154,7 +154,7 @@ public function createDraft($modifierId, $contentTypeId);
      * Copy a Type incl fields and group-relations from a given status to a new Type with status {@link Type::STATUS_DRAFT}.
      *
      * New Content Type will have $userId as creator / modifier, created / modified should be updated, new remoteId created
-     * and identifier should be 'cpy_<identifier>_' + the new remoteId or another unique number.
+     * and identifier should be 'cp_<sourceIdentifier>_<abbreviatedSourceRemoteId>_<newAbbreviatedRemoteId>' or another unique number.
```

Comment:
Good point, in my own defence, it was already like that in `master` :-)
Fixed in eac0467.